x-pack/plugins/alerts/server/task_runner/task_runner.ts

```diff
@@ -442,6 +454,41 @@ function generateNewAndResolvedInstanceEvents(params: GenerateNewAndResolvedInst
   }
 }
 
+function scheduleActionsForResolvedInstances(
+  alertInstancesMap: {
+    [x: string]: AlertInstance<
+      {
+        [x: string]: unknown;
+      },
+      {
+        [x: string]: unknown;
+      }
+    >;
+  },
+  executionHandler: ReturnType<typeof createExecutionHandler>,
+  originalAlertInstanceIds: string[],
+  currentAlertInstances: Dictionary<AlertInstance>,
+  mutedInstanceIds: string[]
+) {
+  const currentAlertInstanceIds = Object.keys(currentAlertInstances);
+  const resolvedIds = without(
+    originalAlertInstanceIds,
+    ...[...currentAlertInstanceIds, ...mutedInstanceIds]
+  );
+  for (const id of resolvedIds) {
+    const instance = alertInstancesMap[id];
+    instance.updateLastScheduledActions(ResolvedActionGroup.id);
+    instance.unscheduleActions();
+    executionHandler({
+      actionGroup: ResolvedActionGroup.id,
+      context: {},
+      state: {},
```

Comment:
Wondering if we're going to find the empty context and state to be a problem 🤔 
Should we perhaps remove the values expected in context and stat from the template variables in the UI?
Otherwise users might try to use them in actions when attaching to Resolved, no?

Reply:
I did it in the UI PR